.github/workflows/docs.yml

```diff
@@ -8,8 +8,6 @@ on:
     paths:
       - 'docs/**'
   workflow_dispatch:
-    paths:
```

Comment:
Is there an advantage to removing this?

Reply:
There is no advantage in having it. It says its no valid